src/main/java/org/alliancegenome/curation_api/interfaces/bulk/ontology/ZfaTermBulkRESTInterface.java

```diff
@@ -0,0 +1,17 @@
+package org.alliancegenome.curation_api.interfaces.bulk.ontology;
+
+import org.alliancegenome.curation_api.base.BaseOntologyTermBulkRESTInterface;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/zfaTerm/bulk")
```

Comment:
All the other endpoints this is lowercase can we make it `zfaterm` to be consistant with the rest of them?